packages/ag-charts-community/src/chart/cartesianChart.ts

```diff
@@ -47,6 +47,14 @@ export class CartesianChart extends Chart {
 
         this.hoverRect = seriesPaddedRect;
 
+        const minRects = this.series.map((series) => (series as CartesianSeries<any, any>).getMinRect?.());
```

Comment:
This could be improved if we passed the series type into Chart<CartesianSeries> perhaps?

Reply:
Maybe add the `getMinRect()` definition to `ISeries` to avoid the need to cast?